public/index.html

```diff
@@ -3,43 +3,46 @@
     <head>
         <meta charset="utf-8">
         <title>Chat</title>
-        <link href="main.css" rel="stylesheet" type="text/css">
+        <link href="app.css" rel="stylesheet" type="text/css">
         <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.1/angular.min.js"></script>
-        <script src="js/main.js"></script>
+        <script type="text/javascript" src="js/app.js"></script>
+        <script type="text/javascript" src="js/controllers/chat.controller.js"></script>
```

Comment:
This is fine for now, but remember that you would want to use a bundler if this was a deployed application. It's up to you whether you use one here, but it is probably a good idea to figure out how they work with switching between a dev and production setup.